website/docs/docs/collaborate/project-recommendations.md

```diff
@@ -4,12 +4,13 @@ sidebar_label: "Project recommendations"
 description: "dbt Explorer provides recommendations that you can take to improve the quality of your dbt project."
 ---
  
-dbt Explorer provides recommendations about your project from the `dbt_project_evaluator` [package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) using metadata from the Discovery API.
+dbt Explorer provides recommendations about your project from the `dbt_project_evaluator` [package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) using metadata from the Discovery API. 
```

Comment:
Hiya @mirnawong1 

This reads really well IMO. A possible suggestion and non-blocker - could Discovery API be hyperlinked to the Discovery API page? (https://docs.getdbt.com/docs/dbt-cloud-apis/discovery-api)

Kind Regards
Natalie

Reply:
good shout! yes will add!